BlockchainSdk/Extensions/BlockchainUtils/Bech32.swift

```diff
@@ -14,6 +14,9 @@ import Foundation
 
 /// Bech32 checksum implementation
 public class Bech32 {
+    
+    private static let BECH32M_CONST = UInt32(0x2bc830a3)
+    private static let BECH32_CONST = UInt32(1)
```

Comment:
```suggestion
    private static let bech32mConst = UInt32(0x2bc830a3)
    private static let bech32Const = UInt32(1)
```
в идеале хоть какой нибудь коммент, почему так, откуда взялось

Reply:
Ну коммент думаю не нужен, потому что это по стандарту taproot адреса, могу ссылочку закрепить, но в целом это в официальной доке в заголовке указано